scipy/optimize/_differentialevolution.py

```diff
@@ -181,6 +189,54 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     >>> result.x, result.fun
     (array([ 0.,  0.]), 4.4408920985006262e-16)
 
+
+    Next example demonstrates the parallelization capabilities.
+
+    >>>
+    from scipy.optimize import rosen
+    from scipy.optimize import differential_evolution as de
+    from scipy.misc import SPool
+    from scipy.misc import PPool
+    import time
+
+    def objfunc(params):
+        for it in xrange(1000000):
```

Comment:
Fixed in https://github.com/pavelponomarev/scipy/commit/d1591124a494da81d87bbaee62d687ffa55c59b6
and in
https://github.com/pavelponomarev/scipy/commit/9a227d00aeff4deafa6068f1e3c11c3f90993a85